enhancements/sandboxed-containers/sandboxed-containers-tech-preview.md

```diff
@@ -0,0 +1,437 @@
+   the machine-os-content.
+
+
+For additional details about RHCOS extensions in general in: https://github.com/openshift/enhancements/pull/317
+
+##### Use RHCOS extensions for QEMU and its dependencies, copy Kata Containers files onto the host
+
+In this option we completely rely on RHCOS Extensions for the QEMU and its
+dependencies, but still build a container imagine containing:
+* Kata Containers binaries:
+  - This is not present on RHEL, but the RPMs are built using the internal
+    Red Hat infrastructure, and then decompressed into the container image.
+
+**Pros:**
+1. The most size costly RPMs can be installed only by the users who would be
+   interested in using the Sandboxed Containers runtimes;
+2. There's no extra work related to properly loading QEMU dependencies, as all
+   the packages would be in installed in the same PATH its build was intent
+   for;
+3. QEMU RPMs have been tested / validated using an identical environment;
+4. No additional work with regarding to re-packaging those in another form
+   other than the RPMs which is the way they are consumed by other Layered
+   Products;
+5. We rely on an already existing mechanism to install part of the RPMs, QEMU
+   and its dependencies, which are not part of the base image.
+
+**Cons:**
+1. 20MB extra - coming from QEMU and its dependencies - as part of the RHCOS
+   Extensions, for all users relying on extensions;
+2. 100MB extra - coming from Kata Containers binaries - copied onto the host,
+   for those who'd be using the Sandboxed Containers;
+3. Part of the RPM installation is still handled by the Sandboxed Containers
+   operator, which may lead to all sorts of issues related to parallel
+   installing packages (in case any other operator decides to do the same);
+4. Updates / Removal may be more complicated than having the RPMs as part of
+   the machine-os-content.
+
+
+##### Use RHCOS extensions for QEMU, QEMU dependencies, and Kata Containers RPMS
+
+In this option we completely rely on RHCOS Extensions for all the RPMs needed.
+
+**Pros:**
+1. There's no extra work related to properly loading QEMU dependencies, as all
+   the packages would be in installed in the same PATH its build was intent
+   for;
+2. QEMU RPMs have been tested / validated using an identical environment;
+3. No additional work with regarding to re-packaging those in another form
+   other than the RPMs which is the way they are consumed by other Layered
+   Products;
+4. We rely on an already existing mechanism to install all of the RPMs - QEMU,
+   QEMU dependencies, and Kata Containers - which are not part of the base
+   image;
+5. There's no chance of hitting issues related to parallel installing packages,
+   as the whole package installation is handled by the MCO;
+6. Updates / Removal become trivial as they are also handled by the MCO.
+
+**Cons:**
+1. 120MB extra - coming from QEMU, QEMU dependencies, and Kata Containers - as
```

Comment:
We can capture the rpm size in the payload for qemu-kiwi and kata as it should be smaller like we discussed.